docs/install/advanced-configuration.rst

```diff
@@ -732,230 +741,841 @@ You can use the following command to test your connection settings:
+
+    *If you have an integer code : int*
+
+    *If you have an string code : str*
+
+|
+
+::
+
+    BLADE_CODE_FORMAT = "{signagecode}-{bladenumber}"
+
+Correspond to the format of blades. Show N3-1 for the blade 1 of the signage N3.
+
+    *If you want to change : move information under bracket*
+
+    *You can also remove one element between bracket*
+
+    *You can do for exemple :*
+    *"CD99.{signagecode}.{bladenumber}"*
+
+    *It will display : CD99.XIDNZEIU.01 (first blade of XIDNZEIU)*
+
+    * *signagecode is the code of the signage*
+    * *bladenumber is the number of the blade*
+
+|
+
+::
+
+    LINE_CODE_FORMAT = "{signagecode}-{bladenumber}-{linenumber}"
+
+Correspond to the format used in export of lines. Used in csv of signage.
+
+    *Similar with above*
+    *You can do for example :*
+    *"CD99.{signagecode}-{bladenumber}.{linenumber}"*
+
+    *It will display : CD99.XIDNZEIU-01.02 (second line of the first blade of XIDNZEIU)*
+
+    * *signagecode is the code of the signage*
+    * *bladenumber is the number of the blade*
+    * *linenumber is the number of the line*
+
+
+**Screenshots**
+
+::
+
+    SHOW_SENSITIVE_AREAS_ON_MAP_SCREENSHOT = True
+    SHOW_POIS_ON_MAP_SCREENSHOT = True
+    SHOW_SERVICES_ON_MAP_SCREENSHOT = True
+    SHOW_SIGNAGES_ON_MAP_SCREENSHOT = True
+    SHOW_INFRASTRUCTURES_ON_MAP_SCREENSHOT = True
+
+Show objects on maps of PDF
+
+|
+
+::
+
+    MAP_CAPTURE_SIZE = 800
+
+Size in pixels of the capture.
+
+    *Be careful with your pdfs.*
+    *If you change this value, pdfs will be rendered differently*
+
+
+**Synchro Geotrek-rando**
+
+::
+
+    SYNC_RANDO_ROOT = os.path.join(VAR_DIR, 'data')
+
+Path on your server where the data for Geotrek-rando website will be generated
+
+    *If you want to modify it, do not forget to import os at the top of the file.*
+    *Check* `import Python <https://docs.python.org/3/reference/import.html>`_ *, if you need any information*
+
+::
+
+    THUMBNAIL_COPYRIGHT_FORMAT = ""
+
+Add a thumbnail on every picture for Geotrek-rando
+
+
+    *Example :*
+
+    *"{title}-:-{author}-:-{legend}"*
+
+    *Will display title of the picture, author*
+    *and the legend :*
+    *Puy de Dômes-:-Paul Paul-:-Beautiful sunrise on Puy de Dômes"*
+
+|
+
+::
+
+    THUMBNAIL_COPYRIGHT_SIZE = 15
+
+Size of the thumbnail.
+
+|
+
+::
+
+    TOURISM_INTERSECTION_MARGIN = 500
+
+Distance to which tourist contents, tourist events, treks, pois, services will be displayed
+
+    *This distance can be changed by practice for treks in the admin.*
+
+|
+
+::
+
+    DIVING_INTERSECTION_MARGIN = 500
+
+Distance to which dives will be displayed.
+
+|
+
+::
+
+    TREK_EXPORT_POI_LIST_LIMIT = 14
+
+Limit of the number of pois on treks pdf.
+
+    *14 is already a huge amount of POI, but it's possible to add more*
+
+|
+
+::
+
+    TREK_EXPORT_INFORMATION_DESK_LIST_LIMIT = 2
+
+Limit of the number of information desks on treks pdf.
+
+    *You can put -1 if you want all the information desks*
```

Comment:
How come this is updating doc so much ?

Reply:
Hmm, true, I must have messed up a thing while resolving the merge conflict...

Reply:
It should be good!